Server/mods/deathmatch/logic/CConsoleCommands.cpp

```diff
@@ -60,103 +61,332 @@ static void EndConsoleOutputCapture(CClient* pClient, const SString& strIfNoOutp
     }
 }
 
-bool CConsoleCommands::StartResource(CConsole* pConsole, const char* szArguments, CClient* pClient, CClient* pEchoClient)
+bool CConsoleCommands::StartResource(CConsole* const console, const char* arguments, CClient* const client, CClient* const echo)
 {
-    if (!szArguments || !szArguments[0])
+    if (!arguments || !arguments[0])
     {
-        pEchoClient->SendConsole("* Syntax: start <resource1> <resource2> ...");
-        return false;
+        echo->SendConsole("* Syntax: start <resource> [<resource> ...] or start <directory> [<directory> ...]");
+        return true;
     }
+     
+    std::istringstream       iss(arguments);
+    std::vector<std::string> names((std::istream_iterator<std::string>(iss)), std::istream_iterator<std::string>());
```

Comment:
I would personally extract this into a dedicated function. Somewhere among Utils. What do you think?

Reply:
We already have that functionality. Take a look at the code this change replaced.